src/App.js

```diff
@@ -30,6 +34,10 @@ class App extends Component {
         };
     }
 
+    onCloseSnackbar = () => {
+        this.props.acSnackbarClosed();
```

Comment:
Almost all the actions we have are named with the format ac + action + thing.
I would rename this to _acCloseSnackbar_.